models/cart-discount/src/cart-discount-draft/presets/sample-data-fashion/employee-sale.ts

```diff
@@ -0,0 +1,33 @@
+import * as CartDiscountValueRelative from '@commercetools-test-data/cart-discount-value-relative';
+import { LocalizedString } from '@commercetools-test-data/commons';
+import {
+  CustomerGroupDraft,
+  TCustomerGroupDraft,
+} from '@commercetools-test-data/customer-group';
+import * as CartDiscountDraft from '../..';
+import { TCartDiscountDraftBuilder } from '../../../types';
+
+const customerGroupDraft = CustomerGroupDraft.presets.sampleDataFashion
+  .employee()
+  .build<TCustomerGroupDraft>();
+
+const employeeSale = (): TCartDiscountDraftBuilder =>
+  CartDiscountDraft.presets
+    .empty()
+    .value(
+      CartDiscountValueRelative.CartDiscountValueRelativeDraft.random()
+        .type('relative')
+        .permyriad(1500)
+    )
+    .cartPredicate(`customer.customerGroup.key = "${customerGroupDraft.key}"`)
+    // TODO: create `target` model
```

Comment:
The remaining work is to create models for `target`, but I wanted to get this PR up as it's already growing quite large